src/main/java/com/faforever/client/fx/DualStringListCellController.java

```diff
@@ -21,6 +27,19 @@ public void setLeftText(String apply) {
     left.setText(apply);
   }
 
+  public void setLeftTextTooltip(String apply) {
+    final WebView web = new WebView();
+    uiService.registerWebView(web);
```

Comment:
u need to do a null check on apply... also u need to remove the tooltip on null because the controller might be reused